src/components/Contacts/ContactDetails/ContactTasksTab/ContactTasksTab.tsx

```diff
@@ -90,10 +84,14 @@ export const ContactTasksTab: React.FC<ContactTasksTabProps> = ({
   accountListId,
   contactId,
 }) => {
-  const { data, loading } = useContactTasksTabQuery({
+  const { data, loading, refetch } = useContactTasksTabQuery({
     variables: { accountListId, contactId },
   });
 
+  const handleSearchTermChanged = (searchTerm: string) => {
+    refetch({ accountListId, contactId, searchTerm });
+  };
```

Comment:
You shouldn't need to call `refetch`. The `useQuery` hook should see that its variables have changed  on rerender and refetch for you. It must use the `useEffect` dependency array under the hood or some sort of change detection.

Reply:
Okay, should I put `searchTerm` inside `useState` and use this method to update the state?